plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2069,6 +2083,84 @@ public void testSplitPruningForFilterOnPartitionColumn()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testAllAvailableTypes()
+    {
+        assertUpdate("CREATE TABLE test_all_types (" +
+                " bool BOOLEAN" +
+                ", int_ INTEGER" +
+                ", big BIGINT" +
+                ", rl REAL" +
+                ", dbl DOUBLE" +
+                ", dec_ DECIMAL(5,2)" +
+                ", dt DATE" +
+                ", time_ TIMESTAMP(6)" +
```

Comment:
time

Reply:
do we support different precisions?

Reply:
according to `io.trino.plugin.iceberg.IcebergPageSink#getIcebergValue` we only support TIME(6). Nice catch tough, thanks